src/omitBy.ts

```diff
@@ -19,15 +19,19 @@ type PickSymbolKeys<T extends object> = {
 // would be omitted from the output, so we need to assume any of them could be
 // filtered out. This means that we effectively make all (enumerable) keys
 // optional.
-type PartialEnumerableKeys<T extends object> = Simplify<
-  IfBoundedRecord<
-    T,
-    PickSymbolKeys<T> & {
-      -readonly [P in keyof T as P extends symbol ? never : P]?: Required<T>[P];
-    },
-    ReconstructedRecord<T>
-  >
->;
+type PartialEnumerableKeys<T extends object> = T extends unknown
```

Comment:
**nit:** whenever we have to do the `T extends unknown` trick, we should add the distributive conditional comment, i think? see definition for `IsBoundedKey`

Reply:
Sure, I will add the distributive conditional comment, it will be useful for future reference!